ui/.env.remote

```diff
@@ -0,0 +1 @@
+CS_URL=http://10.0.32.234:8080
```

Comment:
The .env.remote was an accident, I will remove.
The package-lock.json file should be updated at times , right? It will not build without it, it seems. No problem to keep it out of scope here.

Reply:
the changes on package-lock.json is huge
```
32,591 additions, 440 deletions not shown because the diff is too large. Please use a local Git client to view these changes.
```
Do we need the new nodejs modules ?